arctic/chunkstore/passthrough_chunker.py

```diff
@@ -0,0 +1,75 @@
+from pandas import DataFrame, Series
+
+from ._chunker import Chunker, START, END
+
+
+class PassthroughChunker(Chunker):
+    TYPE = 'passthru'
+
+    def to_chunks(self, df, **kwargs):
+        """
+        pass thru chunker of the dataframe/series
+
+        returns
+        -------
+        ('NA', 'NA', 'NA', dataframe/series)
+        """
+        if len(df) > 0:
+            yield b'NA', b'NA', b'NA', df
+
+    def to_range(self, start, end):
+        """
+        returns a RangeObject from start/end sentinels.
+
+        returns
+        -------
+        string
+        """
+        return b'NA'
+
+    def chunk_to_str(self, chunk_id):
+        """
+        Converts parts of a chunk range (start or end) to a string
+
+        returns
+        -------
+        string
+        """
+        return b'NA'
+
+    def to_mongo(self, range_obj):
+        """
+        returns mongo query against range object.
+        since range object is not valid, returns empty dict
```

Comment:
yes, thats fine. there is only one "chunk" per symbol, so matching on symbol alone is fine. the range keys (start/end) arent necessary or used really.